File: 0412-fizz-buzz/0412-fizz-buzz.cpp
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> resps;
        for (int i = 1; i <= n; i++) {
            string resp;
            if (i % 3 == 0) {resp += "Fizz";}
            if (i % 5 == 0) {resp += "Buzz";}
            if (i%5 != 0 && i%3 != 0) {resp += to_string(i);}
            resps.push_back(resp);
        }
        return resps;
    }
};

File: 0412-fizz-buzz/0412-fizz-buzz.py
class Solution(object):
    def fizzBuzz(self, n):
        ans = [""]*n
        for i in range(n):
            flag = True
            if (i+1) % 3 == 0:
                ans[i] += "Fizz"
                flag = False
            if (i+1) % 5 == 0:
                ans[i] += "Buzz"
                flag = False
            if flag:
                ans[i] += str(i+1)
        return ans 
        """
        :type n: int
        :rtype: List[str]
        """
        